theano/gpuarray/opt.py

```diff
@@ -2143,6 +2158,18 @@ def local_gpu_svd(op, context_name, inputs, outputs):
         return op(inputs[0].astype('float32')).astype('float16')
     return op
 
+
+@register_opt('ctc', 'fast_compile')
+@op_lifter([theano.tensor.nnet.ctc.ConnectionistTemporalClassification])
+@register_opt2([ConnectionistTemporalClassification], 'ctc', 'fast_compile')
+def local_gpu_ctc(op, context_name, inputs, outputs):
+    if not config.ctc.enabled:
+        return
```

Comment:
I'm not sure this check is useful, because if we get here them we already use CTC on the cpu.